tests/test_providers.py

```
@@ -0,0 +1,259 @@
+    def setUp(self):
+        self.faker = Faker()
+        self.created_identifier = None
+
+    def tearDown(self):
+        if self.created_identifier:
+            try:
+                self.scalekit_client.actions.providers.delete_custom_provider(
+                    DeleteCustomProviderRequest(identifier=self.created_identifier)
+                )
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)